base/applications/games/solitaire/solitaire.cpp

```diff
@@ -178,6 +179,16 @@ void SetPlayTimer(void)
     }
 }
 
+void EnableUndoMenu(void)
+{
+    EnableMenuItem(hGameMenu, IDM_GAME_UNDO, MF_BYCOMMAND | MF_ENABLED);
+}
+
+void DisableUndoMenu(void)
+{
+    EnableMenuItem(hGameMenu, IDM_GAME_UNDO, MF_BYCOMMAND | MF_DISABLED | MF_GRAYED);
+}
```

Comment:
You can merge both these functions into one where a BOOL parameter can switch between enable and disable.

Reply:
Done!